cdap-client/src/main/java/co/cask/cdap/client/MetricsClient.java

```diff
@@ -266,6 +266,16 @@ public void waitFor(String name, long count,
         doWaitFor(name, count, timeout, timeoutUnit);
       }
 
+      @Override
+      public long getCounter(String name) {
```

Comment:
Why is the implementation the same for `getCounter` and `getGauge` in `MetricsClient`, but the implementations are different in `RuntimeStats`?